android/Omnivore/app/src/main/java/app/omnivore/omnivore/ui/components/LabelChip.kt

```diff
@@ -0,0 +1,33 @@
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.selection.toggleable
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import app.omnivore.omnivore.ui.components.LabelChipColors
+
+@Composable
+fun LabelChip(
```

Comment:
There's a few chip styles we could use other than the SuggestionChip I had in here: 

https://m3.material.io/components/chips/guidelines#295a6e5e-a1c1-42db-a3f2-fca3f103838a

(but maybe not quite what we want anyways)

Reply:
Yeah I was trying to make those work and none seemed quite right, figured just building our own box would make it easiest to match the web.